ui/src/main/js/store/actions/distributions.js

```diff
@@ -194,7 +199,7 @@ export function fetchDistributionJobs() {
                             jobs.forEach((jobConfig) => {
                                 dispatch(fetchAuditInfoForJob(jobConfig));
                             });
```

Comment:
Create a ticket to do this more efficiently by passing all of the relevant job ids.